tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -119,37 +112,37 @@ def save(self, key_value_pairs) -> None:
         except Exception as e:
             self.alert(str(e))
 
-    def __setitem__(self, key: ShelveKey, val: bytes) -> None:
+    def __setitem__(self, key: str, val: bytes) -> None:
```

Comment:
does the caller know that they must no longer send `ShelveKey` objects here?

Reply:
statemanager is the caller and it calls build_key to obtain keys? So it shouldn't have knowledge about ShelveKey at the first place. It was ShelveKey because the caller was mirror_state_store.